source/_components/ebusd.markdown

```diff
@@ -20,6 +20,9 @@ Enable the sensor by adding the following to your `configuration.yaml` file:
 ebusd:
   host: 127.0.0.1
   circuit: '700'
+  monitored_conditions:
```

Comment:
Only required variables in the first example

Reply:
Thanks. Removed.